src/utils.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+const github = require('@actions/github');
+
+async function createIssue(token, issueTitle, issueBody) {
+  const octokit = github.getOctokit(token);
+
+  return octokit.issues.create({
+    ...github.context.repo,
+    title: issueTitle,
+    body: issueBody,
+  });
+}
+
+async function getLastOpenPendingIssue(token, latestReleaseDate) {
```

Comment:
This logic load the existing issue doesn't look very robust. We should probably use the title of the issue instead of the author. The author is very generic here and I guess another action may have created an issue, meaning that we may be modifying an existing, unrelated issue. Let's use the issue title.

Reply:
The API doesn't allow us to use the issue title. There is no parameter called title which we can pass to the API. 
Maybe what we can do is set a `label` when creating the issue and then search for the issue using that label to update it. What do you think?

Reply:
Ah interesting. Yes let's use something we can customize, a label then. does `notify-release` sound like a sensible label name?

Reply:
Yep. `notify-release` should be good. I will make the changes